hub-show

```diff
@@ -1,275 +1,238 @@
-#!/bin/bash -e
-# Copyright (c) 2022 EPAM Systems, Inc.
+#!/bin/sh -e
+# Copyright (c) 2023 EPAM Systems, Inc.
 #
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# file, You can obtain one at https://www.mozilla.org/en-US/MPL/2.0/
 
-# shellcheck disable=SC2086
+VERBOSE=${VERBOSE:-"false"}
+if test "$VERBOSE" = "true"; then
+  set -x
+fi
 
-HUB_HOME="${HUB_HOME:-$(dirname "$0")}"
```

Comment:
Without `HUB_HOME/bin` in `PATH` utilities like `color`, `dotenv`, etc will not work because of `not found` error.

Reply:
Thanks... reverted `HUB_HOME`. This means `HUB_HOME/bin` place where `dotenv` and other utilities are will be resolved